### PR TITLE
CIRC-270 Add new properties to the item representation on request

### PR DESCRIPTION
**Pull request by Anatolii-Starkov**
-documented Request.item.enumeration field

### CODE REVIEW

File: ramls/request.json [hunk 1]
```diff
@@ -123,6 +123,9 @@
           "type": "string"
         }
       },
+      "enumeration": {
```
Comment (marcjohnson-kint) [inline ramls/request.json]: Properties should have a description, please add one to this property

Comment (Anatolii-Starkov) [reply ramls/request.json]: Hmm, I don't know what it means and there is not a description in item's schema  too 😄

Comment (marcjohnson-kint) [reply ramls/request.json]: I asked Charlotte, and I suggest we use `the volume or issue number for the item`

Comment (Anatolii-Starkov) [reply ramls/request.json]: Done

**Pull request merged**